client.go

```diff
@@ -302,7 +302,7 @@ func (c *client) removeGroupMemberships(ctx context.Context, servicePrincipalObj
 	var merr *multierror.Error
 
 	for _, id := range groupIDs {
-		if _, err := c.provider.RemoveGroupMember(ctx, servicePrincipalObjectID, id); err != nil {
+		if _, err := c.provider.RemoveGroupMember(ctx, id, servicePrincipalObjectID); err != nil {
```

Comment:
ref to RemoveGroupMember - https://github.com/hashicorp/vault-plugin-secrets-azure/blob/6544951c66d6c23b467877757d5f58dc7741dace/provider.go#L41